src/placesview.cpp

```diff
@@ -172,7 +172,7 @@ void PlacesView::onClicked(const QModelIndex& index) {
 
 void PlacesView::setCurrentPath(Fm::FilePath path) {
     currentPath_ = std::move(path);
-    if(path) {
+    if(currentPath_) {
         // TODO: search for item with the path in model_ and select it.
```

Comment:
There are more examples of this in the code and all of them are OK because the move c-ctor of FilePath doesn't free any resource. However, IMO, not using the moved-from object (with some exceptions) seems safer and better for code readability.